src/txkube/test/test_network.py

```diff
@@ -421,7 +423,10 @@ def f1():
 
 
 @attr.s
-class Redirectable(proxyForInterface(IReactorSSL)):
+class Redirectable(proxyForInterface(IReactorSSL),
+                   proxyForInterface(IReactorTCP),
+                   proxyForInterface(IReactorTime),
+                   proxyForInterface(IReactorPluggableNameResolver)):
     """
     An ``IReactorSSL`` which ignores the requested destination and always
```

Comment:
I suppose this part of  the docstring should be updated now.  Maybe just "A reactor ..."?

Reply:
Fixed